nonCisMaleInCs/package.json

```
@@ -19,11 +19,17 @@
     "@react-native-community/eslint-config": "^2.0.0",
     "babel-jest": "^26.6.3",
     "eslint": "^7.12.1",
+    "husky": "^4.3.5",
     "jest": "^26.6.3",
     "metro-react-native-babel-preset": "^0.63.0",
     "react-test-renderer": "16.13.1"
   },
   "jest": {
     "preset": "react-native"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "node run-script.js"
```

```suggestion
      "pre-commit": "npm test"
```
Because our strategy will be running custom tests to ensure the user has added the correct components

---

Done!